utils/node_local_test.py

```diff
@@ -475,6 +491,9 @@ def start(self, clean=True):
         cmd = [daos_server, '--config={}'.format(self._yaml_file.name),
                'start', '-t', '4', '--insecure', '-d', self.agent_dir]
 
+        if self.conf.args.no_root:
+            cmd.append('--recreate-superblocks')
```

Comment:
can you remind me why we are reinstating this?

Reply:
In this case it's to run daos on docker, and in docker you can only specify mounts on the docker run line, so daos server (or any other process we run inside the container) simply doesn't have permission to either mount or re-mount /mnt/daos, hence the need to reinstate this.  Without it then daos_engine tries to umount and then fails.

The upshot of this however is that it can build, run and smoke-test daos within five minutes of pushing a PR on the docker nodes, down from a minimum of 35 minutes currently, and often more depending on scheduling.